data/src/languages_update.py

```diff
@@ -10,14 +10,23 @@
 import os
 import unicodedataplus
 
-from typing import Tuple, Union, DefaultDict, Dict
+from typing import (
+    Dict,
+    DefaultDict,
+    Tuple,
+    Union
+)
 
-from data.src.codes import LANGUAGES_PATH, TSV_DIRECTORY_PATH
+from data.src.codes import (
+    LANGUAGES_PATH,
+    TSV_DIRECTORY_PATH,
+)
 
 
 def _detect_best_script_name(
```

Comment:
An afterthought: Do you think whether the following may be more efficient:

1. Put all the characters of all the words in a single set.
2. Fill the mapping from characters to script names.
3. Pass the mapping to `_detect_best_script_name` and use the character as a key to get the script. This may, in theory, be more efficient than calling into `unicodedataplus`.

WDYT?

Reply:
In theory, it makes sense that it would be a bit more efficient, but I think we're talking about seconds saved, as opposed to minutes if we're updating outside of `split.py` as we are now. On my machine, it takes `languages_update.py` a minute or two max to run, so I would have to log the runtimes to see if we're actually more efficient at this point, but I think the suggested approach would definitely be better if we kept `_detect_best_script_name` in `split.py.`